src/include/distributed/worker_transaction.h

```diff
@@ -24,7 +24,8 @@ typedef enum TargetWorkerSet
 {
 	WORKERS_WITH_METADATA,
 	OTHER_WORKERS,
-	ALL_WORKERS
+	ALL_NON_COORDINATOR_WORKERS,
```

Comment:
Maybe ALL_NON_COORDINATOR_NODES? And possibly similar rename for `WORKERS_WITH_METADATA` to stay consistent.

Reply:
#4015